package.json

```diff
@@ -31,7 +32,8 @@
     "reflect-metadata": "^0.1.13",
     "rimraf": "^3.0.2",
     "rxjs": "^6.5.5",
-    "swagger-ui-express": "^4.1.4"
+    "swagger-ui-express": "^4.1.4",
+    "mongoose": "^5.9.15"
```

Comment:
Een (klein) dingetje wat ik net nog tegen kwam: In de IDE hebben we nu geen intellisense voor Mongoose, dus geen type checken op models. Wil je nog @types/mongoose toevoegen? Zou ik erg blij van worden 😃
`npm install --save-dev @types/mongoose`

Reply:
zal m toevoegen (denk als devdependency toch) :)

Reply:
Yes, dev dependency inderdaad!